src/modules/px4iofirmware/px4io.h

```diff
@@ -159,7 +159,12 @@ extern output_limit_t pwm_limit;
 /*
  * GPIO handling.
  */
+/* HEX Cube Orange and Cube Yellow uses an inverted signal to control the IMU heater */
+#ifdef CONFIG_ARCH_BOARD_HEX_IO_V2
+#define LED_BLUE(_s)			px4_arch_gpiowrite(GPIO_LED1, (_s))
```

Comment:
Could we make this explicit with hex_io-v2 not even having GPIO_LED1 and LED_BLUE?